src/main/java/com/aca/homework/week17/note/service/core/NoteService.java

```diff
@@ -0,0 +1,10 @@
+package com.aca.homework.week17.note.service.core;
+
+import com.aca.homework.week17.note.entity.Note;
+import com.aca.homework.week17.note.service.impl.NoteCreationParams;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
there is no need to annotate the service interface here. you can do that in ServiceImpl class